skins/laika/src/components/pages/Supporters.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 	<div class="supporters">
-		<supporter v-for="( supporter, index ) in supporters"
+		<supporter lang="de" v-for="( supporter, index ) in supporters"
```

Comment:
that kind of goes in the opposite direction of the whole i18n thing we got set up.

Reply:
It was mainly for the `hyphens:auto` CSS attribute to ensure that it will hyphenate the German company names and German donor names according to German hyphenation rules. Just a quirky on feature to make it look better in the browsers that support it.